device-identifier/shbdeviceidentifier/commands.py

```diff
@@ -40,16 +41,26 @@ def run_rpc_server(db: Database):
         db.stop_InfluxDB()
 
 
-def read(db: Database, file_path: click.Path, file_type: str, measurement: str = "main"):
+def read(
+        db: Database, file_path: click.Path, file_type: str, measurement: str = "main", countermeasure_key: str = None
```

Comment:
I suggest we decouple read and countermeasures. For me, the workflow would look better when we perform the countermeasures on data that is already in influxdb. This would require an additional db query. Then we could write the manipulated data back to the db, with some kind of indicator that it was updated (e.g., new measurement or other bucket).